CONTRIBUTOR_ROLES.md

```diff
@@ -0,0 +1,205 @@
+
+### Maintainer requirements
+
+- Experience as an invited expert for at least six months.
+- Demonstrates a broad knowledge of the project across multiple areas.
+- Is able to exercise judgment for the good of the project, independent of the influence of other members.
+- A history of mentoring other contributors.
+- Can commit to spending at least 16 hours per month working on the project.
+- Attends the community meeting which takes place once every two months.
+
+#### Process for nominating a Maintainer
+
+- The nominator will open a pull request using the appropriate [template](https://github.com/mdn/mdn-community/roles/) against the [MDN Web Docs community repository](https://github.com/mdn/mdn-community/).
+- The nominee will add a comment to the pull request agreeing to all responsibilities of becoming a Maintainer.
+- At least three current Maintainers must approve the pull request.
+- Once the pull request is approved, the new Maintainer is added to the [maintainers team](https://github.com/orgs/mdn/teams/maintainers) and any other relevant topic or project teams.
+
+### Maintainer responsibilities
+
+In addition to the responsibilities of all other roles, a Maintainer has these additional responsibilities:
+
+- Determining priorities for the project.
+- Participating in community meetings.
+- Mentoring new and existing contributors across all other roles.
+- If appropriate, based on the skill set of the maintainer:
+  - Propose, approve, or implement code and infrastructure improvements.
+  - Propose, approve, or implement content improvements.
+  - Propose, approve, or implement process improvements.
+
+## Community Manager
+
+The Community Manager role is distinct in many respects. Still, as the projects are technical, the role shares many of the same requirements and responsibilities as the Maintainer role.
+
+In addition to the responsibilities of a Maintainer, the Community Manager has these additional responsibilities.
+
+- Along with Maintainers, address reports where the code of conduct has been violated and decide on the appropriate action.
+- Organize and run community events.
+- Determine media strategies to promote the project and onboard new contributors and users.
+- Define and implement the onboarding experience.
+- Organize community-related project meetings.
+- Meet with and ensure a healthy relationship with contributors and partners.
+- Assist with issue triage as well as pull request review where appropriate.
+- Responsible for monitoring all communication channels, including discussions on GitHub.
+- Highlight contributors that have done exceptional work and/or have shown dedication to the project and organization.
+- Ensure the health and well-being of the project and all participants.
+- Identify and assist with the implementation of automation to improve project sustainability.
+
```

Comment:
Add owner in here - doesn't have to say much, just that it's core team folk

Reply:
Also I wonder whether this file should be at least linked to (if not mirrored) in contribution docs